setup.py

```diff
@@ -6,6 +6,13 @@
 from setuptools import setup
 from setuptools import find_packages
 
+
+if not hasattr(sys, 'real_prefix'):
+    exit('Only supported on a Virtual Environment')
```

Comment:
Unfortunately, this will break a lot of distro packaging efforts, where virtualenv is clearly not necessary (distros often have their own chroot-like mechanisms for package building).